src/APIFunctions/3DPrinting.js

```diff
@@ -72,7 +82,7 @@ export async function getAll3DPrintRequests () {
  * @returns {boolean} - If the request was sucessfully deleted or not.
```

Comment:
Be sure to update the documentation of this comment, we don't return a boolean anymore

```suggestion
 * @returns {ApiResponse} Containing any error information related to the 
 * request
```